DESCRIPTION

```diff
@@ -23,9 +24,11 @@ Imports: glmnet,
     stringi,
     Matrix,
     stringdist,
-    hrbrthemes
+    hrbrthemes,
+    magrittr,
+    purrr,
+    e1071
```

Comment:
Was in relation with some code I have not pushed. But I am not ready at all, I will move it back to suggest.